bundles/org.openhab.binding.pushsafer/src/main/resources/OH-INF/i18n/pushsafer_de.properties

```diff
@@ -12,6 +12,10 @@ thing-type.pushsafer.pushsafer-account.description = Ermöglicht den Zugriff auf
 
 thing-type.config.pushsafer.pushsafer-account.answer.label = Antwort
 thing-type.config.pushsafer.pushsafer-account.answer.description = true \= Aktiviere Antworten auf Push-Benachrichtigungen, sonst false.
+thing-type.config.pushsafer.pushsafer-account.answeroptions.label = Antwort Optionen
```

Comment:
Please revert changes to this file, only default trranslation is provided by PR's all other languages are provided by crowdin.